### PR TITLE
Update package.json for git repo

### PR DESCRIPTION
**Pull request by jeonghonah**
## Proposed changes

- To show git repository in the npm website, `repository` field  need to be updated.
- Added following fields related to the git repo: `repository, bugs, homepage`

## Types of changes

- [ ] Bugfix
- [ ] New feature or enhancement
- [x] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/caver-js/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/caver-js)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,6 +33,14 @@
...
     "klaytn api"
   ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/klaytn/caver-js.git"
+  },
+  "bugs": {
+    "url": "https://github.com/klaytn/caver-js/issues"
+  },
+  "homepage": "https://github.com/klaytn/caver-js#readme",
```
Comment (jimni1222) [inline package.json]: The PR for cdn includes the contents of the repository and homepage, and I would like to match the format.

Does homepage have to be linked to readme file?
Should the repository also have git+ in front of url?

Comment (jeonghonah) [reply package.json]: I think we can change those format. It was just default template when creating a npm package.

Comment (jimni1222) [inline package.json]:
```suggestion
  "homepage": "https://github.com/klaytn/caver-js",
```

File: package.json [hunk 2]
```diff
@@ -33,6 +33,14 @@
     "klaytn sdk",
     "klaytn api"
   ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/klaytn/caver-js.git"
```
Comment (jimni1222) [inline package.json]:
```suggestion
    "url": "https://github.com/klaytn/caver-js.git"
```

File: package.json (hunk 1 above)
Comment (jimni1222) [reply package.json]: Okay cool :) then i will comment with suggested change, can you check?

**Pull request merged**